src/main/java/bitbucketpullrequestbuilder/bitbucketpullrequestbuilder/BitbucketRepository.java

```diff
@@ -158,16 +158,25 @@ private BitbucketCause createCause(AbstractPullrequest pullRequest) {
         String repoUri = null;
 
         if(links != null) {
-            final List<AbstractPullrequest.RepositoryLink> cloneLinks = links.getClone();
+            final List<AbstractPullrequest.RepositoryLink> cloneLinks = links.getCloneLinks();
```

Comment:
Did you miss this one in your  original PR?

Reply:
In the original PR you commented that the original field name is confusing (https://github.com/nishio-dens/bitbucket-pullrequest-builder-plugin/pull/202#discussion_r304996964), this is just the result of the rename.

Reply:
👍